parser/parse.go

```diff
@@ -458,6 +458,7 @@ func (b *Builder) findTypesIn(pkgPath importPathString, u *types.Universe) error
 	for _, f := range b.parsed[pkgPath] {
 		if strings.HasSuffix(f.name, "/doc.go") {
 			tp := u.Package(string(pkgPath))
+			tp.Comments = []string{}
```

Comment:
Why do we need this line? You can append to "nil" slice, so I don't fully understand this line.